examples/handlers/multipart/src/main.rs

```diff
@@ -0,0 +1,126 @@
+//! An example of decoding requests from an HTML form element
```

Comment:
Let's also specify "multipart" in here to make it clear how it's different from the other form example.

Reply:
@colinbankier You can modify it.